forecasting/tests/unittests/models/common.py

```diff
@@ -0,0 +1,40 @@
+"""Common utils and data for all model tests"""
+import random
+
+import pandas as pd
+from gluonts.dataset.common import ListDataset
+
+DUMMY_DATASET = ListDataset(
+    [
+        {
+            "target": [random.random() for _ in range(10)],
+            "start": pd.Timestamp("2022-01-01 00:00:00"),  # noqa
+            "item_id": 0,
+        },
+        {
+            "target": [random.random() for _ in range(10)],
+            "start": pd.Timestamp("2022-01-01 00:00:00"),  # noqa
+            "item_id": 1,
+        },
+    ],
+    freq="H",
+)
```

Comment:
What about a unit test related to converting DataFrame to GluonTS format? Will that be in a later PR?

Reply:
Yes, this doesn't include our dataset implementation yet.